app/controllers/api_controller.rb

```diff
@@ -223,7 +223,25 @@ def validate_include_params
   end
 
   def included_relationships
-    IncludeParamHandler.new(params).call
+    @included_relationships ||= IncludeParamHandler.new(params).call
+  end
+
+  def included_db_relationships
+    @included_db_relationships ||= included_relationships.map { |value| cast_relationship_to_symbol_hash(value) } if included_relationships.present?
+  end
+
+  def cast_relationship_to_symbol_hash(value)
+    parts = value.split('.', 2)
+    if parts.length == 1
+      # special hack to allow for the flags table renamed in the backend but not in the FE includes
```

Comment:
can this comment be updated as this is isn't a rename but rather the choice of how we surface relationships in the API by omitting `framework_` from any PER resources